Formula/gcc.rb

```diff
@@ -76,15 +74,10 @@ def install
     # GCC will suffer build errors if forced to use a particular linker.
     ENV.delete "LD"
 
-    if build.with? "all-languages"
-      # Everything but Ada, which requires a pre-existing GCC Ada compiler
-      # (gnat) to bootstrap. GCC 4.6.0 adds go as a language option, but it is
-      # currently only compilable on Linux.
-      languages = %w[c c++ objc obj-c++ fortran java jit]
-    else
-      # C, C++, ObjC compilers are always built
-      languages = %w[c c++ objc obj-c++]
-    end
+    # Everything but Ada, which requires a pre-existing GCC Ada compiler
+    # (gnat) to bootstrap. GCC 4.6.0 adds go as a language option, but it is
+    # currently only compilable on Linux.
+    languages = %w[c c++ objc obj-c++ fortran java jit]
```

Comment:
```suggestion
    languages = %w[c c++ objc obj-c++ fortran]
```
Let's match Homebrew/core: https://github.com/Homebrew/homebrew-core/blob/39422ff1cc96830687fc4641b37be5091f58ae97/Formula/gcc.rb#L56